packages/react/fast-components-styles-msft/src/utilities/color/neutral-outline-contrast.ts

```diff
@@ -0,0 +1,50 @@
+import { subtract } from "@microsoft/fast-jss-utilities";
+import {
+    neutralOutlineActiveDelta,
+    neutralOutlineFocusDelta,
+    neutralOutlineHoverDelta,
+    neutralOutlineRestDelta,
+    neutralPalette,
+} from "../design-system";
+import {
+    ColorRecipe,
+    colorRecipeFactory,
+    SwatchFamily,
+    SwatchFamilyResolver,
+    swatchFamilyToSwatchRecipeFactory,
+    SwatchFamilyType,
+    SwatchRecipe,
+} from "./common";
+
+import { accessibleAlgorithm } from "./accessible-recipe";
+
+export const neutralOutlineContrastAlgorithm: SwatchFamilyResolver = colorRecipeFactory(
+    accessibleAlgorithm(
+        neutralPalette,
+        3,
+        0,
+        subtract(neutralOutlineHoverDelta, neutralOutlineRestDelta),
+        subtract(neutralOutlineActiveDelta, neutralOutlineRestDelta),
+        subtract(neutralOutlineFocusDelta, neutralOutlineRestDelta)
+    )
+);
+
+export const neutralOutlineContrast: ColorRecipe<SwatchFamily> = colorRecipeFactory(
+    neutralOutlineContrastAlgorithm
+);
+export const neutralOutlineContrastRest: SwatchRecipe = swatchFamilyToSwatchRecipeFactory(
+    SwatchFamilyType.rest,
+    neutralOutlineContrast
+);
+export const neutralOutlineContrastHover: SwatchRecipe = swatchFamilyToSwatchRecipeFactory(
+    SwatchFamilyType.hover,
+    neutralOutlineContrast
+);
+export const neutralOutlineContrastActive: SwatchRecipe = swatchFamilyToSwatchRecipeFactory(
+    SwatchFamilyType.active,
+    neutralOutlineContrast
+);
+export const neutralOutlineContrastFocus: SwatchRecipe = swatchFamilyToSwatchRecipeFactory(
+    SwatchFamilyType.focus,
+    neutralOutlineContrast
+);
```

Comment:
@nicholasrice assuming we'll need to update the `fast-components` implementation with a similar change to meet contrast - or did you take these changes when you ported?